src/ert/_c_wrappers/enkf/_config_content_as_dict.py

```diff
@@ -118,18 +116,13 @@ def config_content_as_dict(
             job_name, args = parse_signature_job("".join(model))
             parsed_jobs.append([job_name, args])
         content_dict[ConfigKeys.FORWARD_MODEL] = parsed_jobs
-    # Add the defines if they exits
-    defines = []
-    if isinstance(site_config_content, ConfigContent):
-        defines += [
-            [key, val] for key, val in site_config_content.get_const_define_list()
-        ]
+
     if isinstance(user_config_content, ConfigContent):
-        defines += [
+        defines = [
             [key, val] for key, val in user_config_content.get_const_define_list()
         ]
-    if defines:
```

Comment:
Some logic unexpectedly changed here. What happened?

Reply:
To minimize code duplication, I decided to use the read_site_config function (which was previously only used by everest), also in ert_configs from_file path. For this reason I use the as_dict twice, once for the site config, and once for the user config. The changed logic is trying to handle reading the two config contents at the same time, and those paths should no longer be hit. Unsure what to put in the commit body.

Reply:
It could be its own commit: "Use read_site_config when reading res_config from file".

Reply:
The refactor is so tied to this change, it won't make it clearer if I even manage to make a separate commit.